src/Fizzics/toolbox.js

```diff
@@ -27,6 +27,7 @@ var FizzicsToolbox = GObject.registerClass(class FizzicsToolbox extends Toolbox
                 new GLib.Variant('(sava{sv})', ['reset', [], {}]),
                 null, Gio.DBusCallFlags.NONE, -1, null
             );
+            this._controlPanel._level2._onNotify();
```

Comment:
This is a bit punching a hole through the different layers...
The `FizzicsLevel2` class already has access to the model `reset` signal through the `bindModel()` method. I think that would be a much better place to connect to `reset` and call `_onNotify()` from.

Reply:
But the 'reset' signal is emitted by the Toolbox class. Not from the model.

Reply:
And I have no access to the Toolbox from FizzicsLevel2.